tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
     "sourceMap": true,
+    "esModuleInterop": true,
```

Comment:
Was this something you added manually or if was done automatically by a schematic?

Reply:
No this is done by the ng update command.